docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -191,6 +191,7 @@ Experimental features might be changed or removed without prior notice.
 | `failWrongDSUID`                            | Throws an error if a datasource has an invalid UIDs                                                                                                                                                                                                                               |
 | `databaseReadReplica`                       | Use a read replica for some database queries.                                                                                                                                                                                                                                     |
 | `alertingApiServer`                         | Register Alerting APIs with the K8s API server                                                                                                                                                                                                                                    |
+| `dashboardRestoreUI`                        | Enables the frontend to be able to restore a recently deleted dashboard                                                                                                                                                                                                           |
```

Comment:
in line 187: Can you please add (backend only) in the description?

Reply:
I'll add it in `registry.go` as the rest of the files are auto generated 👌